woocommerce/woocommerce-android
# [Migrate Simple Payments] Custom amount click opens custom amount section

Closes: #11202


### Description

The PR adds an opening of addition of a custom amount screen from the bottom sheet

### Testing instructions

* More -> Payments -> Collect Payment -> Add a custom Amount
* Notice that the custom amount screen is opened

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/3824d9b6-d2a8-4fcd-bf73-cfd9520886f3
